paasta_tools/cli/cmds/logs.py

```diff
@@ -801,6 +806,8 @@ def determine_scribereader_envs(self, components, cluster):
             # If a component has a 'source_env', we use that
             # otherwise we lookup what scribe env is associated with a given cluster
             env = LOG_COMPONENTS[component].get('source_env', self.cluster_to_scribe_env(cluster))
+            if 'additional_source_envs' in LOG_COMPONENTS[component]:
```

Comment:
I guess here would be the place for some sort of test. `test_determine_scribereader_envs_handles_additional_envs`